utils/build/docker/cpp/parametric/.clang-format

```diff
@@ -0,0 +1,6 @@
+---
+Language: Cpp
+ColumnLimit: 180
+DerivePointerAlignment: false
+PointerAlignment: Left
```

Comment:
Do whatever you like here.

I notice, though, that elsewhere we base the style on Google's. This config is implicitly based on LLVM's. Do you like this look better?